integration-tests/easy-rag/src/test/resources/__snapshots__/EasyRagCodestartTest/testContent/src_main_java_ilove_quark_us_Bot.java

```diff
@@ -0,0 +1,34 @@
+package ilove.quark.us;
+
+import dev.langchain4j.service.SystemMessage;
+import dev.langchain4j.service.UserMessage;
+import io.quarkiverse.langchain4j.RegisterAiService;
+import jakarta.enterprise.context.SessionScoped;
+
+/**
+ * This is a sample Bot, it is configured to ingest the 'easy-rag-catalog/'.
+ * You can @Inject this Bot in your Rest resource
+ *
+ * <code>
+ *     @Inject
+ *     Bot bot;
+ *
+ *     @POST
+ *     @Produces(MediaType.TEXT_PLAIN)
+ *     public String chat(String q) {
+ *         return bot.chat(q);
+ *     }
+ * </code>
```

Comment:
This isn't the proper way to add code in Javadoc. Let's not complicate things and just remove it completely.

Reply:
Ok, I've fixed the doc. 

This is also done in other codestarts, let's be consistant.

Reply:
Just updated the correponding snapshot